NOTICE

```diff
@@ -47,6 +44,106 @@ types-urllib3 1.26.15 : Apache License 2.0
+	Copyright (c) 2017-2021 Ingy d
+	Copyright (c) 2006-2016 Kirill Simonov
+Pyflakes 2.4.0 pypi:pyflakes/2.4.0
+	Copyright 2005-2011 Divmod, Inc.
+	Copyright 2013 Florent Xicluna.  See LICENSE file for detailsfrom __future__ import with_statement
+	Copyright 2013-2014 Florent Xicluna
+RonnyPfannschmidt/iniconfig 1.1.1 pypi:iniconfig/1.1.1
+	(C) Ronny Pfannschmidt, Holger Krekel -- MIT licensed"""__all__ = ['IniConfig', 'ParseError']
+atomicwrites 1.4.0 pypi:atomicwrites/1.4.0
+	copyright = '2015, Markus Unterwaditzer
+	Copyright (c) 2015-2016 Markus Unterwaditzer
+	copyright = '2015-2017, Markus Unterwaditzer
+charset-normalizer 2.0.12 pypi:charset-normalizer/2.0.12
+	Copyright (c) 2019 TAHRI Ahmed R.
+	copyright: (c) 2021 by Ahmed TAHRI
+	copyright = '2019, Ahmed TAHRIauthor = 'Ahmed TAHRI @Ousret'
+dataclasses 0.8 pypi:dataclasses/0.8
+	(C)        with self.assertRaisesRegex(TypeError,            asdict(int)
+	(C)        with self.assertRaisesRegex(TypeError,            astuple(int)
+	(C)        # the_fields is a tuple of 3 items, each value        #  is in __annotations__.        self.assertIsInstance(the_fields, tuple)        for f in the_fields:
+	(C)        with self.assertRaisesRegex(TypeError,            fields(C())
+flake8 4.0.1 pypi:flake8/4.0.1
+	Copyright (C) 2011-2013 Tarek Ziade <tarek@ziade.org>
+	(C) Copyright ..." is shown in the HTML footer. Default is True.
+	Copyright (C) 2012-2016 Ian Cordasco <graffatcolmingov@gmail.com>
+	copyright = True
+importlib-metadata 4.2.0 pypi:importlib-metadata/4.2.0
+	Copyright 2017-2019 Jason R. Coombs, Barry Warsaw
+isort 5.10.1 pypi:isort/5.10.1
+	Copyright (c) 2021 Taneli Hukkinen
+	Copyright (c) 2013 Timothy Edmund Crosley
+mccabe 0.6.1 pypi:mccabe/0.6.1
+	Copyright © 2011-2013 Tarek Ziade <tarek@ziade.org>
+	Copyright © 2013 Florent Xicluna <florent.xicluna@gmail.com>
+	Copyright © <year> Ned Batchelder
+mypy-extensions 0.4.3 pypi:mypy-extensions/0.4.3
+	Copyright (c) 2016-2017 Jukka Lehtosalo and contributors
+pathspec 0.9.0 pypi:pathspec/0.9.0
+	Copyright 2019-2020 Kevin Locke <kevin@kevinlocke.name>"""Script to reinstall pip before running `pip install`.
+	Copyright © 2013-2021 Caleb P. Burns"
+py 1.11.0 pypi:py/1.11.0
+	(C) Ronny Pfannschmidt, Holger Krekel -- MIT licensed"""__all__ = ['IniConfig', 'ParseError']
+	(C) Copyright ..." is shown in the HTML footer. Default is True.
+	copyright = True
+	copyright: This stylesheet has been placed in the public domain.
+pycodestyle 2.8.0 pypi:pycodestyle/2.8.0
+	(C) Copyright ..." is shown in the HTML footer. Default is True.
+	Copyright (C) 2009-2014 Florent Xicluna <florent.xicluna@gmail.com>
+	copyright = True
+	Copyright © 2009-2014 Florent Xicluna <florent.xicluna@gmail.com>
+	Copyright © 2014-2020 Ian Lee <IanLee1521@gmail.com>
+	Copyright (C) 2014-2016 Ian Lee <ianlee1521@gmail.com>
+	Copyright (C) 2006-2009 Johann C. Rocholl <johann@rocholl.net>
+	Copyright © 2006-2009 Johann C. Rocholl <johann@rocholl.net>

```

Comment:
Contains a couple of improper copyright attributions.

Reply:
Fixed.